zerver/webhooks/lidarr/doc.md

```diff
@@ -0,0 +1,24 @@
+Receive Lidarr notifications in Zulip!
+
+1.  {!create-stream.md!}
+
+1.  {!create-bot-construct-url-indented.md!}
+
+1.  Go to your Lidarr
+    dashboard. Click **Settings** and then **Connect**.
+    Click the **+** icon.
+
+1.  Select **Webhook**. Enter **Zulip** or any other name you wish in the
+    name field. Tick which scenarios you would like to be notified for. You
+    can enter tags if you would only like to notify for artists with a
+    specific tag.
```

Comment:
This would be better phrased like this:

Select **Webhook** and set the name of the webhook to any name of your choice (e.g **Zulip**). Select the scenarios you would like to receive notifications for. You may also enter tags if you would like to be notified about artists with specific tags.